packages/runtime/src/prop-controllers/copy/backgrounds.ts

```diff
@@ -44,21 +46,39 @@ export function copy(
               })),
             },
           }
-
-        case 'image':
+        })
+        .with(
+          [
+            { version: 1 },
+            { type: 'image-v1', version: 1, payload: { image: { type: 'makeswift-file' } } },
```

Comment:
My understanding of this is that copies for external files will include a reference to the same external file.

Reply:
Exactly